src/NServiceBus.CustomChecks.Tests/ApprovalFiles/APIApprovals.Approve.approved.txt

```diff
@@ -30,6 +30,8 @@ namespace NServiceBus
 {
     public static class CustomChecksConfigurationExtensions
     {
+        public static void AddCustomCheck<TCustomCheck>(this NServiceBus.EndpointConfiguration config)
+            where TCustomCheck :  class, NServiceBus.CustomChecks.ICustomCheck { }
         public static void ReportCustomChecksTo(this NServiceBus.EndpointConfiguration config, string serviceControlQueue, System.TimeSpan? timeToLive = default) { }
```

Comment:
Raw thoughts: 

Should the AddCustomCheck method be on the level of ReportCustomChecksTo or on an object that gets returned from ReportCustomChecksTo? It seems AddCustomCheck would not work without ever calling ReportCustomChecksTo. The counterargument might be that we want to allow adding custom checks anywhere.

Reply:
I could see where an architecture team would want to have an INeedInitialization (or similar function) that would want to add a custom check as a concept without being able to specify where they should go.

Alternatively, should there be a startup warning that there are custom checks registered but no destination configured?